www/includes/easyparliament/templates/html/header.php

```diff
@@ -29,8 +29,8 @@
   <?php if ($og_image) { ?>
     <meta property="og:image" content="<?= $og_image ?>">
     <meta property="og:image:type" content="image/png">
-    <meta property="og:image:width" content="1000">
-    <meta property="og:image:height" content="500">
+    <meta property="og:image:width" content="1200">
+    <meta property="og:image:height" content="630">
   <?php } else { ?>
     <meta property="og:image" content="https://<?= DOMAIN ?>/images/og/social_<?= $current_assembly ?? 'uk' ?>.jpg">
     <meta property="og:image:type" content="image/jpeg">
```

Comment:
There is no `social_uk.jpg` image so the extension here should be a png. Some of the social graphics are jpgs and some are png so need to standardise for this to work.

`$current_assembly` does not seem to be consistently set so e.g. the scottish homepage gets the uk parliament link.

Reply:
All of the social graphics *should* be pngs. I've copied over the old London Assembly as a jpg, and I'll just replace that with the UK one as a holder if we end up coming back to the LA.

Reply:
I've added the current_assembly in a few obvious places. It's not the end of the world to fall back to the UK one, but this should catch the obvious ones.